js/ui/scheduler/tooltip_strategies/tooltipStrategyBase.js

```diff
@@ -167,6 +167,7 @@ export class TooltipStrategyBase {
         $container.append($deleteButton);
         this.scheduler._createComponent($deleteButton, Button, {
             icon: "trash",
+            stylingMode: "text",
```

Comment:
Add some tests, please. Check directly, that correct 'stylingMode' is applied to the delete button.